improtreskApi/rest_router.py

```diff
@@ -1,17 +1,33 @@
 from rest_framework import routers
 
-from api.rest import accomodations, lector_roles, lectors, rules, workshops,\
+from api.rest import accomodations, lector_roles, lectors, participants, rules, workshops,\
     workshop_difficulties, years
 from api_textual.rest import news, texts, workshop_locations
 
-router = routers.DefaultRouter()
+from rest_framework_extensions.routers import NestedRouterMixin
+
+
+class NestedDefaultRouter(NestedRouterMixin, routers.DefaultRouter):
+    pass
+
+
+router = NestedDefaultRouter()
 router.register(r'lectorRoles', lector_roles.LectorRoleViewSet)
 router.register(r'lectors', lectors.LectorViewSet)
 router.register(r'news', news.NewsViewSet)
 router.register(r'texts', texts.TextViewSet)
 router.register(r'workshopDifficulties', workshop_difficulties.WorkshopDifficultyViewSet)
 router.register(r'years', years.YearViewSet)
 router.register(
+    r'workshops',
```

Comment:
`/workshops` were removed in favor of /years/x/workshops. Please move this under `/years/x/workshops/y/participants/`